indico/modules/events/tracks/models/tracks.py

```diff
@@ -94,6 +100,14 @@ class Track(DescriptionMixin, db.Model):
         lazy=True,
         backref='default_for_tracks'
     )
+    group = db.relationship(
+        'TrackGroup',
+        lazy=True,
+        backref=db.backref(
+            'tracks',
+            lazy='dynamic'
```

Comment:
do we really need/want `lazy='dynamic'` here?

Reply:
I think I used it here since I wanted to filter the `tracks` but I don't think we are gonna need it so probably I don't need `dynamic` here.